ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.component.js

```diff
@@ -124,10 +128,13 @@ export default class ConfirmAddSuggestedToken extends Component {
               type="default"
               large
               className="page-container__footer-button"
-              onClick={() => {
-                removeSuggestedTokens().then(() =>
-                  history.push(mostRecentOverviewPage),
+              onClick={async () => {
+                await Promise.all(
+                  suggestedAssets.map(async ({ id }) => {
+                    await rejectWatchAsset(id);
+                  }),
```

Comment:
```suggestion
                await Promise.all(
                  suggestedAssets.map(({ id }) => rejectWatchAsset(id)
                ),
```

you don't need to await inside here, the await of Promise.all waits for all returned promises in the array to be resolved.

Reply:
Indeed ser!

Reply:
✅